airbyte-webapp/src/core/domain/catalog/traverseSchemaToField.test.ts

```diff
@@ -2,7 +2,7 @@ import { AirbyteJSONSchema } from "core/jsonSchema/types";
 
 import { traverseSchemaToField } from "./traverseSchemaToField";
 
-describe(`${traverseSchemaToField}`, () => {
+describe(`${traverseSchemaToField.name}`, () => {
```

Comment:
I noticed that the test name for this was spitting out the function definition 😆